pkg/component/runtime/conn_info_server_nonwin_test.go

```diff
@@ -0,0 +1,21 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Can you name this file `conn_info_server_unix_test.go`? We normally use `unix` in the filename when its for non-Windows, and being that this specifically use `net.Dial("unix", ...)` its more correct.